### PR TITLE
CPDTP-304 Allow reset of sandbox data

### PR DESCRIPTION
**Pull request by timabell**
### Context

There are multiple lead providers with the same name in the sandbox. After discussion we think it'll be expedient to reset the whole database and recreate the tokens rather than manually fixup the data.

### Changes proposed in this pull request

* Allow reset of sandbox data

### Review Checks
- [ ] All pages have automated accessibility checks via cypress
- [ ] All pages have visual tests via cypress + percy
- [ ] All `School` queries are correctly scoped - `eligible` when they need to be

### How can I view this in a review app?

### CODE REVIEW

**Review by kicferk1 — APPROVED** [review]
I said it like three times already, buut...

Make sure you can recreate their tokens before you run it on sandbox 😂

File: lib/tasks/safe_reset.rake [hunk 1]
```diff
@@ -3,7 +3,7 @@
 namespace :db do
   desc "Remove all database content, then seed it. Only for dev environments."
   task safe_reset: :environment do
-    if Rails.env.development? || Rails.env.deployed_development?
+    if Rails.env.development? || Rails.env.deployed_development? || Rails.env.sandbox?
```
Comment (ethanmills) [inline lib/tasks/safe_reset.rake:6]: Have you thought twice? 😛

**Review by ethanmills — APPROVED** [review]

File: lib/tasks/safe_reset.rake (hunk 1 above)
Comment (timabell) [reply lib/tasks/safe_reset.rake:6]: Could you elaborate on your concerns? I'm not sure what you're getting at

Comment (kicferk1) [reply lib/tasks/safe_reset.rake:6]: I think it's a joke about the thing that the script prints out :D 

> puts "You should think twice before recreating staging or production database!"
puts "Fire off Rails console and copy the code from this task if you really have to."
puts "Make sure you know what you are doing and maybe do a back-up first."

Comment (timabell) [reply lib/tasks/safe_reset.rake:6]: I haven't had a direct response so I'm going to assume Michal is right and  it's fine and go ahead and merge. We can always take it out again.

**Pull request merged**